verilog/CST/parameters.cc

```diff
@@ -77,6 +77,41 @@ const verible::TokenInfo& GetParameterNameToken(const verible::Symbol& symbol) {
   return ABSL_DIE_IF_NULL(identifier_leaf)->get();
 }
 
+std::vector<const verible::TokenInfo*> GetAllParameterNameTokens(
+    const verible::Symbol& symbol) {
+  //Can be used only in kParamDeclaration context
+  auto t = symbol.Tag();
+  CHECK_EQ(t.kind, verible::SymbolKind::kNode);
+  CHECK_EQ(NodeEnum(t.tag), NodeEnum::kParamDeclaration);
```

Comment:
In general, prefer to use `GetSubtreeAsNode` which combines these together, but this check seems to be covered by `GetParameterNameToken` already.  (Can remove)